scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -297,6 +302,10 @@ def matmat_XH_X(x):
         eigvec, _ = np.linalg.qr(eigvec)
 
     elif solver == 'propack':
+        if not HAS_PROPACK:
+            raise ValueError("`solver='propack'` is opt-in due to potential issues on Windows, "
+                             "it can be enabled by setting the `USE_PROPACK` environment "
+                             "variable before importing scipy")
```

Comment:
Needs an update of the environment variable here too.

Reply:
Thanks, I've pushed that adjustment in. So far, I've decided to ignore the linter complaints though, unless there are strong objections to that.

Reply:
Please fix the line length.  There is no reason to not remain in compliance with such a simple formatting rule.